deploy/sre-prometheus/aws/100-ebs-iops-burstbalance.PrometheusRule.yaml

```diff
@@ -10,15 +10,17 @@ spec:
   groups:
   - name: sre-ebs-iops-burstbalance
     rules:
-    - alert: EbsVolumeBurstBalanceLT40PctSRE
-      expr: avg(ebs_iops_credits) < 40
+    - alert: EbsVolumeBurstBalanceLowSRE
+      expr: 100 - min_over_time(ebs_iops_credits[30m:5m]) > 80
       for: 60m
       labels:
         severity: warning
         namespace: openshift-monitoring
-    - alert: EbsVolumeBurstBalanceLT20PctSRE
-      expr: avg(ebs_iops_credits) < 20
-      for: 60m
+    - alert: EbsVolumeBurstBalanceEmptySRE
+      expr: 100 - min_over_time(ebs_iops_credits[3h:15m]) == 100
+      annotations:
+        message: "The EBS volume {{ $labels.vol_id }} has exhausted its IOPs burst credit for over 2 hours"
+      for: 120m
```

Comment:
What does SRE do when this happens?  If we have this as a critical alert paging SRE we need an SOP.  I don't see that in ops-sop.

Reply:
That's true.
Sadly the existing alert didn't have one either.
I guess we need a card

Reply:
It's hard to know how often this occurs and to what volumes - and I think that's going to be a factor in how we put together a SOP for this. I can try to put together a stub at least for tracing the volume to its workload and general debugging.